github/projects.go

```diff
@@ -332,7 +332,7 @@ type ProjectCardOptions struct {
 	// The ID (not Number) of the Issue or Pull Request to associate with this card.
```

Comment:
I think this line also needs to be updated. It says:

```
// The ID (not Number) of the Issue or Pull Request to associate with this card.
```

But GitHub documentation says:

```
The id of the issue to associate with this card.
```